src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -799,7 +802,11 @@ def enable_zip_deploy(cmd, resource_group_name, name, src, timeout=None, slot=No
         if app_is_linux_webapp and track_status is not None and track_status:
```

Comment:
`logger.warning("Starting zip deployment. This operation can take a while to complete ...")`
Shall we revisit this with newer sequencing?